app/pages/lab/workflow-create-form.cjsx

```diff
@@ -25,6 +25,7 @@ WorkflowCreateForm = createReactClass
     workflowToClone = @props.workflowToClone
 
     newWorkflow =
+      id: workflowToClone?.id
       display_name: @refs.newDisplayName.value
       primary_language: workflowToClone?.primary_language || @props.project.primary_language
```

Comment:
these `primary_language`, `steps`, `grouped` and `prioritized` attributes won't be needed anymore as the API copy function uses the source workflow data vs anything the client passes. I think the only attribute we need here is the `display_name` to update the workflow name after copying.

Reply:
👍🏻